### ISSUE TITLE
USWDS - Banner: Some screen reader setups may not announce the full banner

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds#5926]
# Summary
It's possible for some screen readers to only announce "Here's how you know" instead of the entire banner copy, and this lack of context could be confusing to blind users. It's recommended that we consider solutions to make the banner announcement more useful for screen reader users in forms mode, such as making more of the text part of the interactive button since forms mode only takes users to interactive elements.

# Observations
Upon interacting with the banner component, the JAWS screen read a blind participant was using only announced "Here's how you know" instead of the entire banner copy. We believe they were in "forms mode." It did not hinder the participant or confuse them at all, but it's reasonable to assume this could confuse others.

we were unable to replicate this issue internally, however.

# Affected user groups

- JAWS users

# Research method
Usability testing with 8 participants with disabilities: 
- 5 visually impaired (3 blind, 2 low vision/legally blind (1 with nystagmus))
- 1 motor impairment
- 1 ADHD
- 1 anxiety and depression

Screen reader software participants used: JAWS, NVDA, and VoiceOver

[Link to public findings report forthcoming...here is the internal [Zebra findings report](https://docs.google.com/document/d/1vbGSh4UNZiseyzdGXa1pJwU_3M4y7HaU3QPkXOa7gsI/edit?usp=sharing)] :lock:

# Next steps

Consider solutions that would guarantee the entire banner is announced by screen readers in a multitude of tech setups and in JAWS forms mode.

**Comment by jaclinec** [comment]
I'll bring this to an upcoming CS UX sync to discuss if and how we might add any warnings on the banner component page for this issue.

**Comment by jaclinec** [comment]
We discussed at today's CS/UX sync and agreed that since we've only observed this issue one time and we can't replicate the problem, that no further action is needed right now. We'll close this issue but keep it in mind in future testing. If this issue crops up again, we will reassess.
